blog/_posts/2021-01-15-snap-announcement.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "A Snap package for ROOT on Linux"
+layout: archive
+author: James Carroll
+---
+
+There's a new experimental package format for ROOT, based on the [Snap package manager](https://snapcraft.io) from Canonical. This package can be ideal for new ROOT users, new Linux users, or people who's ROOT requirements might be entirely satisfied with an immutable container image. Take a look at the store listing at [https://snapcraft.io/root-framework](https://snapcraft.io/root-framework), where you'll find installations for some common distributions, E.G, [Ubuntu](https://snapcraft.io/install/root-framework/ubuntu), [Mint](https://snapcraft.io/install/root-framework/mint), [Debian](https://snapcraft.io/install/root-framework/debian), [Fedora](https://snapcraft.io/install/root-framework/fedora), [OpenSUSE](https://snapcraft.io/install/root-framework/opensuse), [Arch](https://snapcraft.io/install/root-framework/arch), and more!
+
+On Ubuntu, it's as simple as:
+```
+sudo snap install root-framework && root
+```
+
+You might be able to even just search for the ROOT Framework and install it in a single click!
+
+![ROOT in the Ubuntu App Store]({{ '/assets/images/Snap_Announcement_01.png' | relative_url }})
+
+This is full-fat installation of ROOT, complete with its utilities such as hadd, PyROOT via Python 3.8 (with SciPy, NumPy, Pandas and Matplotlib), and JupyROOT.
+You'll get these bundled by default, and since the whole package is based on container technology, they don't interfere with any of your system libraries and can be easily upgraded (automatically!), removed, and mixed alongside other ROOT installations.
+
+For example, just run `root` in the terminal after installation and you'll get exactly what you expected.
+Give `root --notebook` a go and try out the JupyROOT support.
+As a special case, if you want PyROOT, you'll need to run `pyroot` rather than `python`; this ensures you get the bundled version of Python in the container rather than the host system, but from there you can import ROOT normally and run your normal scripts, `pyroot -i $(root-config --tutdir)/pyroot/fillrandom.py` away! 
+
+There's no need to mount the $HOME directory, graphical support should work by default, and a lot of optional packages are built by default.
```

Comment:
$HOME -> `$HOME`

Reply:
you'll need to run -> you must run
; this ensures -> . This ensures

Reply:
I've made sure all the references to $HOME are code-formatted, and corrected "This ensures"